loader/src/main/kotlin/net/weavemc/loader/mixin/Services.kt

```diff
@@ -60,8 +60,8 @@ internal class SandboxedMixinService : IMixinService {
     override fun getMixinContainers() = emptyList<IContainerHandle>()
     override fun getResourceAsStream(name: String): InputStream? = javaClass.classLoader.getResourceAsStream(name)
     override fun getSideName() = Constants.SIDE_CLIENT
-    override fun getMinCompatibilityLevel() = null
-    override fun getMaxCompatibilityLevel() = null
+    override fun getMinCompatibilityLevel() = MixinEnvironment.CompatibilityLevel.JAVA_7
+    override fun getMaxCompatibilityLevel() = MixinEnvironment.CompatibilityLevel.MAX_SUPPORTED
```

Comment:
`MAX_SUPPORTED` I believe is Java 13, which means this'll break on modern versions of the game.